src/components/CaptureViewer/CaptureVideoViewer.tsx

```diff
@@ -1,24 +1,33 @@
 import { h, FunctionComponent } from 'preact'
 import classNames from 'classnames'
 import { withBlobPreviewUrl } from './hocs'
+import { trackException } from '../../Tracker'
 import style from './style.scss'
 
 type CaptureVideoViewerProps = {
   ariaLabel: string
   className?: string
   previewUrl?: string
+  onVideoError: () => void
 }
 
 const CaptureVideoViewer: FunctionComponent<CaptureVideoViewerProps> = ({
   ariaLabel,
   className,
   previewUrl,
+  onVideoError,
 }) => (
   <div className={classNames(style.videoWrapper, className)}>
     <video
       aria-label={ariaLabel}
       className={style.video}
       src={previewUrl}
+      onError={(event) => {
```

Comment:
What's the default type of `event` here? Can we explicitly infer its type here to avoid (extra) casting inside the function?

Reply:
In React I think it would have been `SyntheticEvent<HTMLVideoElement>` but Preact does not have Synthetic events so I didn't know of any other way to handle the `property not existing on EventTarget` TS error that was originally appearing without the extra casting.

Reply:
Okay fair enough!